src/reader/TranslatableText.js

```diff
@@ -13,6 +14,9 @@ export function TranslatableText({
   bookmarkToStudy,
   overrideBookmarkHighlightText,
   setIsRendered,
+  boldExpression,
+  exerciseType,
```

Comment:
not today, but we have to create a subclass of `TranslatableText` that is ExerciseText. It does not makes sense to have `exerciseType` in the generic `TranslatableText` which is also used, e.g., in `ArticleReader`. Maybe create pls an issue so we don't forget about it.